include/ginkgo/core/matrix/csr.hpp

```diff
@@ -307,6 +307,19 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
 
     std::unique_ptr<LinOp> conj_transpose() const override;
 
+    /**
+     * Sorts all (value, col_idx) pairs in each row by column index
+     */
+    void sort_by_column_index();
+
+    /*
+     * Tests if all row entry pairs (value, col_idx) are sorted by column index
+     *
+     * @returns True if all row entry pairs (value, col_idx) are sorted by
```

Comment:
`@return` and `@returns` actually do the exact same thing according to the [doxygen documentation](http://doxygen.nl/manual/commands.html#cmdreturn).
I used `returns` here because I thought it would form a better sentence, but if you want it uniformly, I can change it everywhere to `@return`.

Reply:
If they do the same then that's fine.